code/modules/research/stock_parts.dm

```diff
@@ -328,3 +328,94 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 	icon = 'icons/obj/stock_parts.dmi'
 	icon_state = "capacitor"
 	desc = "A debug item for research."
+
+
+//Mysterious machine components made by an unknown source. Designed to be doled out as an ultra-rare loot item.
+/obj/item/stock_parts/capacitor/quantumcap
```

Comment:
apparently these all have missing icons, and also i dont like the idea of T5 parts in general, remind me to remove these